packages/timebar/src/components/timeline.js

```diff
@@ -494,12 +508,24 @@ Timeline.propTypes = {
 }
 
 Timeline.defaultProps = {
+  labels: {
+    dragLabel: 'Drag to change the time range',
+    lastUpdate: 'Last update',
+    bookmark: {
+      goToBookmark: 'Go to your bookmarked time range',
+      deleteBookmark: 'Delete time range bookmark',
+    },
+  },
   bookmarkStart: null,
   bookmarkEnd: null,
   bookmarkPlacement: 'top',
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Same comment as in https://github.com/GlobalFishingWatch/frontend/pull/412/files#diff-8d883c6ad4d5adc2918da62ed6f38b515c9bee18b4bb0b114618afda233cd7aaR233